MVC/view/CLI.py

```diff
@@ -124,81 +202,19 @@ def startGame(self):
-                        self.showHoneyComb()
-                        self.controller.controllerUpdatePuzzleState1()
-                    elif isAuto.lower() == "no":
-                        userInput = input("Choose a pangram: ")
-                        self.check = self.controller.controllerCheckPangram(userInput)
-                        while(self.check == False):
-                             userInput = input("Your pangram doesn't exist within our database, please try again. Enter a pangram: ")
-                             self.check = self.controller.controllerCheckPangram(userInput)
-                        self.controller.controllerRunBaseGame(userInput)
-                        print("User letters: " + self.controller.controllerGetLetters())
-                        print("Required letter: " + self.controller.controllerGetReqLetter())
-                        self.showHoneyComb()
-                        self.controller.controllerUpdatePuzzleState1()
+                    self.newPuzzle()
                 case "showpuzzle":
-                        #lots of these are just printing stuff can be removed, just for testing purpsoe.
-                        if (self.controller.controllerGetPuzzleState() == 0):
-                            print("No game started!")
-                        else:
-                            print("Your letters: " + self.controller.controllerGetLetters())
-                            print("Required letter: " + self.controller.controllerGetReqLetter())
-                            print("Guessed words: " + str(self.controller.controllerGetGuessedWordsCLI()))
-                            self.showHoneyComb()
+                    self.showPuzzle()
                 case "showfoundwords":
-                        if (self.controller.controllerGetPuzzleState() == 0):
-                            print("No game started!")
-                        else:
-                            print("Guessed words: " + str(self.controller.controllerGetGuessedWordsCLI()))
+                    self.showFoundWords()
                 case "shuffleletters":
-                        if (self.controller.controllerGetPuzzleState() == 0):
-                            print("No game started!")
-                        else:
-                            self.controller.controllerShuffleAuto()
-                            self.showHoneyComb()
-                            print("Your letters: " + self.controller.controllerGetLetters())
+                    self.shuffleLetters()
                 case "savepuzzle":
-                        if (self.controller.controllerGetPuzzleState() == 0):
-                            print("No game started!")
-                        else:
-                            inputFile = input("Please enter a name for the file: ")
-                            self.controller.controllerSaveGame(inputFile)
+                    self.savePuzzle()
                 case "loadpuzzle":
```

Comment:
uhh didn't you make a function for this above like the rest of the commands? why is the code still here...